.idea/vcs.xml

```diff
@@ -1,36 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!--
```

Comment:
Please do not change this file, it was used for quick links to GitHub from IDEA.

Reply:
DONE